### ISSUE TITLE
Panic if RC.exe fails

### ISSUE THREAD

**Issue by gabdube** [nabijaczleweli/rust-embed-resource#6]
If the linked .rc file is not valid, RC.exe will choke on it but cargo will continue the build. The result is an error like this one:

LINK : fatal error LNK1181: cannot open input file 'versioninfo.lib'

In my case, the true error is this:

![image](https://user-images.githubusercontent.com/5180925/29148916-71911610-7d3e-11e7-9e29-816ca508356b.png)

Panicking if the exitcode not 0 perhaps? Here's what the code could look like:

```rust
let result = Command::new(find_windows_sdk_rc_exe().as_ref().map_or(Path::new("rc.exe"), Path::new))
        .args(&["/fo", &format!("{}/{}.lib", out_dir, prefix), resource])
        .status()
        .expect("Are you sure you have RC.EXE in your $PATH?");
if !result.success() {
 panic!("RC.exe did not exit successfully")
}
```

**Comment by nabijaczleweli** [comment]
That's a nice improvement and a great codewise suggestion, thanks!

**Comment by nabijaczleweli** [comment]
Released in [v1.1.4](https://github.com/nabijaczleweli/rust-embed-resource/releases/tag/v1.1.4).
